data_plotly_plot.py

```diff
@@ -58,6 +58,80 @@ class Plot(object):
 
     def __init__(self, plot_type, plot_properties, plot_layout):
 
+        # Define default plot dictionnary used as a basis for plot initilization
```

Comment:
good idea to define in advance the dictionary with all the plot and layout properties. This means that also in the future, when adding new plot or layout parameters we have to add them also here, isn't it?